scripts/setup/requirements.esp32.txt

```diff
@@ -9,7 +9,7 @@ reedsolo>=1.5.3,<=1.5.4
 bitarray==2.6.0
 bitstring>=3.1.6,<4
 ecdsa>=0.16.0
-construct==2.10.54
+construct==2.10.70
```

Comment:
maybe we can just say >= for now? It seems odd that we require such exact versions in here ... I am not clear why as all the others are ranges.

Reply:
sounds good to me, done!